english/light-fire/stop-drop-roll/start-licking-yourself/start-licking-yourself.md

```diff
@@ -1,2 +1,14 @@
 
 You are covered in thick marshmallow and stuck to the floor. You make an attempt to lick-your-way-out.
+
+Out of the goop emerges a marshmellow dog. The dog apparently has better licking skills than you. She licks you clean in no time.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.